char_sheet_html.py

```diff
@@ -0,0 +1,311 @@
+
+
+<div class="column">
+<table style="float: left">
+  <thead>
+    <tr>
+    <th style= "font-size:8pt">Character Name</th>
+  </tr>
+  <tr>
+    <td>{{char_name}}</td>
+  </tr>
+  <tr>
+    <th style= "font-size:8pt">Pronouns</th>
+  </tr>
+  <tr>
+    <td>{{char_pronouns}}</td>
+  </tr>
+  <tr>
+    <th style= "font-size:8pt">Role</th>
+  </tr>
+  <tr>
+    <td>{{char_role}}</td>
+  </tr>
+</thead>
+</table>
+</div>
+
+<div class="column">
+<table style="float: left">
+    <thead>
+    <tr>
+    <th>Primary Skill</th>
+    <td>Brute: +1</td>
+    </tr>
+    </thead>
+  <tr>
+    <th>Level</th>
+    <td>{{char_level}}</td>
+  </tr>
+  <tr>
+    <th>AR</th>
+    <td>2</td>
+  </tr>
+</table>
+</div>
+
+<br style="clear: both;">
+
+<table style="float: left; font-size: 12pt">
+    <thead>
+    <tr>
+    {% for attribute in attributes %}
+      <th>{{attribute.name}}</th>
+    {% endfor %}
+    </tr>
+    </thead>
+    <tr>
+    {% for attribute in attributes %}
+      <td>{{attribute.value}}</td>
+    {% endfor %}
+    </tr>
+  </table>
+
+  <br style="clear: both;">
+
+<div class="column">
+<table style="float: left;">
+  <tr>
+    <th>Skills</th>
+    <th>Modifier</th>
+  </tr>
+   {% for skill in skills %}
+  <tr>
+    <td>{{skill.name}}</td>
+    <td>{{skill.value}}</td>
+  </tr>
+  {% endfor %}
+</table>
+
+</div>
+
+<div class="column">
+<table style="float: left">
+  <tr>
+    <th>Stat</th>
+    <th>Current</th>
+    <th>Max</th>
+  </tr>
+  {% for stat in stats %}
+  <tr>
+    <td>{{stat.name}}</td>
+    <td>{{stat.current}}</td>
+    <td>{{stat.max}}</td>
+  </tr>
+  {% endfor %}
+</table>
+</div>
+
+<br>
+
+<table>
+    <tr>
+      <th>Inventory</th>
+      <th>Quantity</th>
+      <th>Info</th>
+    </tr>
+    {% for item in items %}
+    <tr>
+      <td>{{item.name}}</td>
+      <td>{{item.quantity}}</td>
+      <td>{{item.info}}</td>
+    {% endfor %}
+    </tr>
+  </table>
+
+</div>
+
+<br style="clear: both;">
+
+<div>
+
+<table>
+    <tr>
+      <th>Power</th>
+      <th>Type</th>
+      <th>PP</th>
+      <th>Mechanic</th>
+    </tr>
+    {% for power in powers %}
+    <tr>
+        <td>{{power.name}}</td>
+        <td>{{power.type}}</td>
+        <td>{{power.ppcost}}</td>
+        <td>{{power.mechanic}}</td>
+    {% endfor %}
+    </tr>
+  </table>
+
+ <table>
+  <tr>
+    <th>Vulnerability</th>
+    <th>Type</th>
+    <th>Mechanic</th>
+</tr>
+    {% for vulny in vulnys %}
+    <tr>
+        <td>{{vulny.name}}</td>
+        <td>{{vulny.type}}</td>
+        <td>{{vulny.mechanic}}</td>
+    {% endfor %}
+</table>
+
+</div>
+
+<br style="clear: both;">
+
+</body>
+</htmL>
+""")
+
+# render() command requires each variable be called within the template to function correctly
+html = template.render(char_name=char_name, char_pronouns=char_pronouns, char_role=char_role, char_level=char_level, powers=powers, vulnys=vulnys, items=items, attributes=attributes, skills=skills, stats=stats)
```

Comment:
please install the `black` formatter. lmk if you want some help setting it up so that it auto-formats on save